### PR TITLE
FOUR-9327: (QA observation)  The Name does not appear in the top order

### PR DESCRIPTION
**Pull request by CarliPinell**
## Issue & Reproduction Steps
The _Name_ does not appear in the top order,please check the vue
in some view the name has the following structure:

Name
Name_something
Something_Name
Something_Name_Something

## Related Tickets & Packages
Tickets from QA
- FOUR-9327

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:deploy

### CODE REVIEW

File: resources/js/admin/users/components/SecurityLogsModal.vue [hunk 1]
```diff
@@ -223,8 +229,8 @@ export default {
       return auxArraySorted;
     },
     /**
-     * Verify if value is a string o null
```
Comment (pmPaulis) [inline resources/js/admin/users/components/SecurityLogsModal.vue:226]: Please not remove the CS

**Review by pmPaulis — CHANGES_REQUESTED** [review]

File: resources/js/admin/users/components/SecurityLogsModal.vue (hunk 1 above)
Comment (CarliPinell) [reply resources/js/admin/users/components/SecurityLogsModal.vue:226]: OK, that Code was 100% my code that's why I applied CS. I'll be careful with foreign code.

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-fdb45cb341.eng.processmaker.net

File: resources/js/admin/users/components/SecurityLogsModal.vue (hunk 1 above)
Comment (fagubla) [inline resources/js/admin/users/components/SecurityLogsModal.vue:226]: don't remove the space, the code style for comments in JS, the * are aligned vertically
```
/**
 * Verify if value is a string o null
 */
isString(value) {
```

**Review by fagubla — CHANGES_REQUESTED** [review]

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-fdb45cb341.eng.processmaker.net

**Review by fagubla — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-fdb45cb341.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-fdb45cb341.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-fdb45cb341.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-fdb45cb341.eng.processmaker.net

**Pull request merged**